src/pages/PlayReason.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState, useEffect } from "react";
+import { withRouter, RouteComponentProps, useParams } from "react-router-dom";
+import { useCookies } from "react-cookie";
+import * as QueryString from "query-string";
+
+import { Button, Layout, Select, Typography, Radio } from "antd";
+
+import Header from "../components/Header";
+import "../styles/PlayReason.css";
+import TextArea from "antd/lib/input/TextArea";
+
+import { createCourse } from "../types/course";
+import { API_DOMAIN } from "../config";
+
+interface ParamTypes {
+  chartID: string;
+}
+
+const NONE_SELECTED = "A reason for playing the HeaRT Game must be selected";
+const NO_TEXT = "Please type in a reason for playing the HeaRT Game";
+
+const PlayReason: React.FC<RouteComponentProps> = (props) => {
+  const [cookies] = useCookies(["accessToken"]);
+  const { chartID } = useParams<ParamTypes>();
```

Comment:
unused variable? I don't think this or the ParamTypes interface is needed at all then?